pkg/console/operator/operator.go

```diff
@@ -201,6 +205,11 @@ func (c *consoleOperator) handleSync(operatorConfig *operatorsv1.Console, consol
 		return fmt.Errorf("console is in an unknown state: %v", operatorConfigCopy.Spec.ManagementState)
 	}
 
+	sErr := c.handleConfigMapSync(operatorConfig)
```

Comment:
Does this short circuit the loop here?  I think a couple issues could arise at this point.